### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by juliandevelops** [NickRTR-Schule/Vokabeltrainer#23] [Concept & Draft]
Diskussionen über Icons und Bilder in dem Programm

**Comment by juliandevelops** [comment]
Auf dem Dashboard, gibt es die Kategorieübersichtslabel, bei welchen wir hinten einen Pfeil nach unten brauchen, um dies auf- bzw. zuzuklappen. Dies brauchen wir als Icon

**Comment by NickRTR** [comment]
Auf-/Zuklappen Icons:

![grafik](https://user-images.githubusercontent.com/95316598/226849645-80dbd4b4-5541-4781-ac70-ab69f58c9b1c.png)

![grafik](https://user-images.githubusercontent.com/95316598/226849717-6e3db764-13c4-4d01-b1b0-4e79b6e6599a.png)

![grafik](https://user-images.githubusercontent.com/95316598/226849904-f53e54b8-aa52-475a-9f71-c4b8cbec937f.png)

1. https://www.reshot.com/free-svg-icons/item/arrow-down-circle-YLGRHVT4P3/
2. https://www.reshot.com/free-svg-icons/item/chevron-arrow-down-circle-XUEYQV7HKJ/
3. https://www.reshot.com/free-svg-icons/item/arrow-downward-C8XPS7VA9D/

**Reply by NickRTR** [reply]
@Jules-sh @Adrian-Cxll Bitte Abstimmen, welches am besten ist.

**Reply by juliandevelops** [reply]
Eins finde ich am unpassendsten.
2 und 3 passen beide, ich tendiere aber eher zu 2.
Es sei denn wir machen ein ähnliches Icon wie 2 als "Back Button". Dann sollten wir 3 nehmen um genug Kontrast zu behalten

**Comment by juliandevelops** [comment]
Als weiteres Icon benötigen wir noch eines in der NavigationBar um zurück zu gehen

**Reply by NickRTR** [reply]
![grafik](https://user-images.githubusercontent.com/95316598/226983562-2453de52-46b1-4e13-abb1-43fb2382601a.png)
 Was sagst du?

**Reply by Adrian-Cxll** [reply]
Kann sich sehen lassen

**Reply by juliandevelops** [reply]
Sieht gut aus, passt würde ich sagen

**Comment by juliandevelops** [comment]
Wir benötigen noch mehr Icons:
1. Ein Löschen Icon (Eine Papiertonne oder so)
2. Eine art share icon (das soll das öffnen einer einzelnen Vokabel repräsentieren)

**Reply by NickRTR** [reply]
Papiertonne: https://www.reshot.com/free-svg-icons/item/delete-2Y37PEBC9U/
Share: https://www.reshot.com/free-svg-icons/item/share-arrow-WS4HD3PU8X/

Was sagt ihr?

**Reply by juliandevelops** [reply]
Finde ich passen beide gut.
Ich werd bei nächster Gelegenheit mal schauen dass ich sie einbaue
